api/lib/resolvers.js

```diff
@@ -263,6 +244,23 @@ const resolvers = (networkList, notificationController) => ({
             validator.operatorAddress
           ].profile
         : undefined
+    },
+    feed: async (validator, _, { dataSources }) => {
+      const networkList = await db.getNetworks()
+      // get feed for this single validator
+      const validatorWithFeed = await getValidatorFeed(
+        validator,
+        validator.operatorAddress,
+        networkList,
+        remoteFetch(dataSources, validator.networkId),
+        networkList.find(({ id }) => id === validator.networkId)
+      )
+      localStore(dataSources, validator.networkId).validators[
```

Comment:
why are you updating the validator here?

Reply:
ah yeah, makes no sense true